### ISSUE TITLE
Pre-training and meta-training setup issues

### ISSUE THREAD

**Issue by luotan369** [SCLBD/MCG-Blackbox#7]
![QQ_1732005333664](https://github.com/user-attachments/assets/c158970b-0e15-4683-b529-442bdbaa53d4)
Hello, I have encountered a problem with the dataset while trying to reproduce the training process using the CIFAR-10 dataset. During pre-training, do train_dataset_root and valid_dataset_root correspond to the pre-processed CIFAR-10 training and validation sets?Are these two datasets also used during meta-training?If not, how should train_dataset_root and valid_dataset_root be set during pre-training and meta-training?Are support_set and query_set corresponding to the target model and substitute model in the meta-training phase? Additionally, during the pre-training phase, when training the C-Glow model without changing any parameters, is the gradual appearance of negative loss due to the x_size setting?
I'm looking forward to your help, thank you very much！
